Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Isolation of Uldum and Pandaria

## Changelog:
- The Shroud of Uldum and Mists of Pandaria now block interactions with outsiders and vice versa.

## Developer changelog:
- All interactions now have checks whether characters are isolated
- Added on_actions regulating isolation events
- Added illustrations for Uldum and Jade Forest in Pandaria
- Localized events regulating the isolation (copypasted from GoA1 rendition)
- Simplified isolation trigger (thanks Robmart)

## Tests:

# How to test:
- Play as any character in Uldum and Pandaria and verify if they cannot interact with characters beyond.
- Play as any character outside of Uldum and Pandaria and verify if they cannot interact with characters within.
- Wait until 610 for the event where Shroud of Uldum is lifted
- Wait until 613 for the event where Mists of Pandaria are lifted
- Alternatively, write `event isolation.1` or `event isolation.2` in console to manually trigger the events lifting isolation
- Verify if Pandaria and Uldum can interact with outsiders and vice versa
